cirq/testing/__init__.py

```diff
@@ -48,6 +48,9 @@
 from cirq.testing.consistent_qasm import (
     assert_qasm_is_consistent_with_unitary,)
 
+from cirq.testing.consistent_specified_has_unitary import (
+    assert_specifies_has_unitary_if_unitary,)
```

Comment:
Nit: Are all these trailing commas on the same line as the parenthesis intended?

Reply:
it's definitely nice to have them there when you add new methods

Reply:
Yes, intended.